atlas-core/src/main/java/org/atlasapi/output/OutputContentMerger.java

```diff
@@ -732,6 +730,15 @@ private void mergeFilmProperties(Application application, Film chosen,
         chosen.setCertificates(combinedCertificates);
     }
 
+    private <T extends Content> void mergeReleaseDates(T chosen, Iterable<T> orderedContent) {
+        Set<ReleaseDate> combinedReleaseDates = MoreStreams.stream(orderedContent)
+                .map(Content::getReleaseDates)
+                .flatMap(Collection::stream)
+                .collect(Collectors.toSet());
```

Comment:
```suggestion
                .collect(MoreCollectors.toImmutableSet());
```
To preserve order and match existing logic.